### DISCUSSION TITLE
Japanese text in Twitter Card is rendered with Chinese font

### DISCUSSION THREAD

**Discussion by rhysd** [community/community#6161] [Other Feature Feedback, Questions, & Ideas]
Japanese text in Twitter Card is rendered with Chinese font. Many Kanji characters are rendered in Chinese, not Japanese. This looks weird for Japanese users like me. I wonder that detecting a Japanese title by checking Hiragana characters in it would be an option. There may be more excellent language detection algorithm.

Example of the rendered card is as follows:

<img width="612" alt="スクリーンショット 2021-10-10 2 42 55" src="https://user-images.githubusercontent.com/823277/136668903-18d82c6d-18ff-4fd0-a5b3-7f2866035a02.png">

Link to the tweet: https://twitter.com/Reputeless/status/1446885369488629763

**Comment by serima** [comment]
Upvoted. I think that this website will help you understand this issue.
https://heistak.github.io/your-code-displays-japanese-wrong/
